src/primitives/transaction.h

```diff
@@ -319,22 +326,30 @@ class CTransaction
 public:
     typedef boost::array<unsigned char, 64> joinsplit_sig_t;
 
-    // Transactions that include a list of JoinSplits are version 2.
-    static const int32_t MIN_CURRENT_VERSION = 1;
-    static const int32_t MAX_CURRENT_VERSION = 2;
+    // Transactions that include a list of JoinSplits are >= version 2.
+    static const int32_t SPROUT_MIN_CURRENT_VERSION = 1;
```

Comment:
`SPROUT_CURRENT_MIN_VERSION`  or `SPROUT_MIN_VERSION` sound clearer to me than `SPROUT_MIN_CURRENT_VERSION`